test/modules/geometry-calculation.test.js

```diff
@@ -1,18 +1,19 @@
-
 const { calculateDistance, calculateJunctionPoint, isPointOnLine } = require('../../modules/geometry-calculation')
 const Point = require('../../modules/ecs6-class/point')
 const Line = require('../../modules/ecs6-class/line')
+const mockLine = jest.fn(constructor)
+const mockPoint = jest.fn(constructor)
```

Comment:
this is not how to mock a class, only a constructor
you could  have written only one mockFunction for the constructor,
it could have been used for all constructors: line, point and more

Reply:
I search a lot of how to mock constructor and didn't find, what to do?

Reply:
mock a class ES6,
not a constructor 
What did you write in your presentation about mocks? which example did you give? 
change your presentation according to what  you've learned